sdk/mobile-center/src/test/java/com/microsoft/azure/mobile/MobileCenterTest.java

```diff
@@ -164,7 +164,7 @@ public void useDummyServiceTest() {
         DummyService service = DummyService.getInstance();
         assertTrue(MobileCenter.getInstance().getServices().contains(service));
         verify(service).getLogFactories();
-        verify(service).onChannelReady(any(Context.class), notNull(Channel.class));
+        verify(service).onStarted(any(Context.class), eq(DUMMY_APP_SECRET), notNull(Channel.class));
```

Comment:
Will rollback to any, the @NotNull is already enforcing that check at compile time